### DISCUSSION TITLE
使用key查询不稳定

### DISCUSSION THREAD

**Discussion by wroldchinalee** [apache/doris#7168] [A - General / Q&A]
使用三台虚拟机，配置8核64G内存，大概70亿数据，表模型为unique，按照key查询时间不稳定，请问有什么优化措施吗

**Comment by morningman** [comment]
The unqiue key model use `merge-on-read` logic to read the data.
Which means the latency of query depends on the compaction progress of data file.

Are there actual metrics showing how unstable it is?
